src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,37 @@
+@import "../utils/variables";
+
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, $card-width);
+  grid-template-rows: repeat(2, $card-heigth);
+  grid-auto-flow: row;
+  grid-auto-rows: $card-heigth;
+  gap: 48px;
+  justify-content: center;
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+  }
+}
+
+@media screen and (min-width: 768px) and (max-width: 1023px) {
+  .catalog {
+    grid-template-columns: repeat(3, $card-width);
+  }
+}
+
+@media screen and (min-width: 488px) and (max-width: 767px) {
+  .catalog {
+    grid-template-columns: repeat(2, $card-width);
+  }
+}
+
+@media screen and (min-width: 0) and (max-width: 487px) {
+  .catalog {
+    grid-template-columns: repeat(1, $card-width);
+  }
+}
```

Comment:
use scss nesting and check your media because you miss pixels 
![image](https://user-images.githubusercontent.com/97547824/202908342-f1939410-7720-4f36-8e43-96aa43fd3934.png)